web/styles/input_pill.css

```diff
@@ -191,6 +192,32 @@ meant to show profile cards when clicked */
     }
 }
 
+.stream_creator_details > .display_only_pill.inline_with_text_pill {
+    padding: 0;
+
+    > .pill {
+        margin: 0;
+        align-items: baseline;
+
+        > .pill-image {
+            /* Add line-height equal to height to mimic baseline alignment. */
+            line-height: 20px;
+            align-self: center;
+        }
+
+        > .pill-label {
+            > .pill-value {
+                padding: 0 5px;
+                max-width: none;
+            }
+
+            > .my_user_status {
+                margin-right: 2px;
+            }
+        }
+    }
+}
```

Comment:
I'm not sure what the cause is, but the pills sometimes show blue link styling on hover -- can you debug that?

Reply:
I have reproduced this in dark mode. Probably the best solution here is to use `.display_only_pill` selector instead of `.user_panel_list > .pill_container` in `dark_theme.css`.

I'll resolve this in a cleanup #frontend thread with @sujalshah-bit as suggested above in this PR.